src/assets/icons/icon-trash-dark.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+  <g fill="#32363f" fill-rule="evenodd" transform="translate(-96 -32)">
```

Comment:
Something that worked so far but I didn't get the chance to make it a rule is to fill the SVG with the `currentColor` keyword and then then change the `color` property of the element wrapping the SVG **BUT ONLY** when we're talking about inline SVGs, not in CSS background :P